pkg/cmd/bazci/bazci.go

```diff
@@ -90,6 +98,10 @@ func bazciImpl(cmd *cobra.Command, args []string) error {
 	}
 	bepLoc := filepath.Join(tmpDir, "beplog")
 	args = append(args, fmt.Sprintf("--build_event_binary_file=%s", bepLoc))
+
+	besEnv := getRunningEnvForBes(strings.TrimPrefix(os.Getenv("TC_BUILD_BRANCH"), "refs/heads/"))
+	args = append(args, fmt.Sprintf("--bes_backend=%s", besBackendHost))
```

Comment:
We'll have to update this PR to use BEP logs from a file, right?

Reply:
Indeed, I am working on changes to this PR and on a beaver hub PR and expect to have both ready by end of today